core/src/components/badge/test/a11y/badge.e2e.ts

```diff
@@ -23,3 +24,27 @@ configs({ directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
     });
   });
 });
+
+configs({ directions: ['ltr'], themes: ['light', 'dark'] }).forEach(({ config, title }) => {
+  test.describe(title('badge: a11y'), () => {
+    test('should not have accessibility violations', async ({ page }) => {
+      /**
+       * All page content should be contained by landmarks (main, nav, etc.)
+       * By containing the badge in a main element, we can avoid this violation.
+       */
+      await page.setContent(
+        `
+        <main>
+          <ion-badge>123</ion-badge>
+          <ion-badge class="ion-color">123</ion-badge>
```

Comment:
Why do we add `ion-color` here? `ion-badge` should use the primary color palette by default.

Reply:
Ah, this was accidentally left in. Thank you for catching it!